javascript/ql/src/semmle/javascript/GlobalAccessPaths.qll

```diff
@@ -440,6 +439,7 @@ module AccessPath {
      *
      * Only has a result if there exists both a read and write of the access-path within `bb`.
      */
+    pragma[nomagic]
```

Comment:
This prevented a catastrophic join-order when I did the "remove all `cached` annotations"-experiment.  
I don't know if it has an effect with the current implementation, but I'm keeping it anyway.